sidebars.js

```diff
@@ -57,6 +57,7 @@ const sidebars = {
             'reference-guides/architecture/deployment',
           ]
         },
+      'reference-guides/rancher-turtles-chart/values',
```

Comment:
I’m not sure how this looks like in the output documentation. May need to be added to the `Reference Guides` items.